.github/workflows/oui_filepull.yml

```
@@ -11,6 +11,8 @@ jobs:
     env:
       BRANCH_NAME: "OUI_Updates"
     steps:
+      - name: "Delete existing branch"
+        run: "git branch -D $BRANCH_NAME"
```

```suggestion
        run: "git branch -D $BRANCH_NAME || true"
```
@abates mentioned that if the branch happened to not exist this would cause a non 0 exit code which would kill the pipeline.  The `|| true` will make it so that it always returns a 0 exit code (true)